arbos/arbosState/arbosstate.go

```diff
@@ -296,7 +296,16 @@ func (state *ArbosState) UpgradeArbosVersion(
 					ErrFatalNodeOutOfDate,
 				)
 			}
-			// no state changes needed
+			// Update the PerBatchGasCost to a more accurate value compared to the old v6 default.
+			ensure(state.l1PricingState.SetPerBatchGasCost(l1pricing.InitialPerBatchGasCostV12))
```

Comment:
We could alternatively just initialize new chains with the new default, but it makes the upgrade simpler to not require a separate parameter change in addition

Reply:
agreed